### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD024.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#29] [General]
# https://blog.3place.cn/3place/fsd/FSD024.html

https://blog.3place.cn/3place/fsd/FSD024.html

**Comment by picklecai** [comment]
我们何须缩短路程！——

在曲折的溪谷中盘桓，
然后攀登这座山岩，
听山泉永无休止地滚滚奔腾，
一路上大大提高我们的游兴！
桦树枝头已闹着春意，
枞树也感到阳春来到；

难道春光没有刺激我们的肢体？

浮士德：随时随地，在生活。
这是蹉跎一辈子之后的痛彻领悟吧？

**Comment by picklecai** [comment]
那些悬崖弯腰曲背，
还有那长长的岩鼻，
都在打鼾，都在透气！

行人希望，和他一起赶路的，都有生命，都是陪伴。

**Comment by picklecai** [comment]
我已爬了三百年之久，
还不能够爬上峰顶

我已等待了千年，
为何城门还不开？

**Comment by picklecai** [comment]
瓦尔普吉斯之夜原是英国土著督伊德教教徒的古老节日，后改为圣瓦尔普吉斯的瞻礼日。在每年的4月30号和5月1日，教徒点燃篝火并跳舞来庆祝寒冬的逝去，迎接春天的来临。由于督伊德教在基督教徒看来是异教，其神被认为是魔鬼。还由于督伊德教徒越来越少，他们常在夜间秘密举行仪式，到圣山献祭品并燃烧五月篝火，于是产生了魔鬼在布罗肯山聚会的迷信传说。而虔诚的瓦尔普吉斯的名字从此和魔鬼连在了一起，“瓦尔普吉斯之夜”也成为魔鬼狂欢节的代用语。因此“瓦尔普吉斯之夜”又被称为“魔女之夜”。

**Comment by picklecai** [comment]
我在前行，我来领你进去，
我重新替你做个介绍。

我在前边开路，领你进去，
帮你重新结识一位美娇娘。

果然玛格蕾特翻篇了

**Comment by picklecai** [comment]
谁能对国民寄予信任！
尽管对他们立过大功；
因为民众就像妇人，
他们总是将青年吹捧。

民众总是非理性的

**Comment by picklecai** [comment]
具有稳健、审慎的内容的书本，
现在究竟有谁爱看！
至于谈到那些年轻的人，
从没有像他们那样自命不凡。

原来哪个时代都有审核与叛逆

**Comment by picklecai** [comment]
没有一柄剑，没有弃誓背盟
而从背后突然结果对方的性命。

越好的剑，用的人越多，概率就越大。
只有不成材的朽木，才能保障自己的道德品质。

**Comment by picklecai** [comment]
人山人海拚命往上挤，
你想向前挤，你却被人所挤。

整个人流一齐往上涌，
你以为在挤人，实际被人挤。

**Comment by picklecai** [comment]
我在这里非常愉快，
跟他们一起很开心；
因为我从恶魔的存在
推论出一定有善灵。

看见面包圈，推测有面包。
看见影子，推测有光亮。
